XIVSlothCombo/Combos/RDM.cs

```diff
@@ -643,56 +656,258 @@ const int
     internal class SimpleRedMage : CustomCombo
     {
         protected internal override CustomComboPreset Preset { get; } = CustomComboPreset.SimpleRedMage;
+
+        internal static bool inOpener = false;
+        internal static bool openerFinished = false;
+        internal static byte step = 0;
+
         protected override uint Invoke(uint actionID, uint lastComboMove, float comboTime, byte level)
 
         {
-            if (actionID == RDM.Veraero || actionID == RDM.Verthunder || actionID == RDM.Verstone || actionID == RDM.Verfire || actionID == RDM.Riposte || actionID == RDM.Zwerchhau || actionID == RDM.Redoublement)
+            if (actionID == RDM.Jolt || actionID == RDM.Veraero || actionID == RDM.Verthunder || 
+                actionID == RDM.Verstone || actionID == RDM.Verfire || actionID == RDM.Riposte || 
+                actionID == RDM.Zwerchhau || actionID == RDM.Redoublement)
```

Comment:
I think in C# you can use the ```is A or B or C``` macro instead of chaining ```== || ==```

Reply:
Yep , didnt want to change it too much from the original , the macro would be better, going to change it.